arguments/SubgroupOps.argdown

```diff
@@ -40,6 +47,7 @@ Expose a subset of subgroup operations in WGSL shaders:
   - <Quadgroup Extension>: If quad operations are made into their own
                            extension, both its potential market becomes
                            larger and subgroup operations' market grows.
+    +> <Viable>
```

Comment:
could you clarify this link for me? does this statement make the original motivation more viable?
I'm confused, because now there appears to be a loop. Motivation -> operation, and operation -> viable

Reply:
Oops, that was meant to go under stage section